src/components/Modals/TeleportModal.vue

```diff
@@ -53,6 +60,20 @@ export default defineComponent({
     rightButtonIsDisabled: { type: Boolean, default: false },
     isSimpleModal: { type: Boolean, default: false }, // true if the modal will set its own styling for its position
     hasNoBackground: { type: Boolean, default: false }, // true for modals without the gray overlay behind them
+    hasClickableTransparentBackground: { type: Boolean, default: false },
+    hasCustomPosition: { type: Boolean, default: false },
+    position: { type: Object as PropType<{ x: number; y: number }>, default: { x: 0, y: 0 } },
```

Comment:
Nit: Like the other props, can you add comments for what these are for?

Reply:
Would also be useful to add them to the notion writeup! https://www.notion.so/courseplan/Modal-Refactor-fbbea3dfda52468db4719f53df7d9464

Reply:
done!